pkg/frontend/clustermanager_type_test.go

```diff
@@ -20,127 +20,128 @@ func TestValidateOcmResourceType(t *testing.T) {
 			err:                  "the resource type 'machinepool' is not valid for api version '2022-09-04'",
 		},
 		{
-			name: "syncidentityprovider - resource type is valid",
+			name:            "syncidentityprovider - resource type is valid",
+			ocmResourceType: "syncidentityprovider",
+			apiVersion:      "2022-09-04",
 			vars: map[string]string{
-				"ocmResourceType":           "syncidentityprovider",
-				"api-version":               "2022-09-04",
 				"resourceProviderNamespace": "microsoft.redhatopenshift",
 			},
 			syncIdentityProviderConverter: mock_api.NewMockSyncIdentityProviderConverter(controller),
 			err:                           "",
 		},
 		{
-			name: "syncidentityprovider - resource type is invalid",
+			name:            "syncidentityprovider - resource type is invalid",
+			ocmResourceType: "invalid",
+			apiVersion:      "2022-09-04",
 			vars: map[string]string{
-				"ocmResourceType":           "invalid",
-				"api-version":               "2022-09-04",
 				"resourceProviderNamespace": "microsoft.redhatopenshift",
 			},
 			syncIdentityProviderConverter: mock_api.NewMockSyncIdentityProviderConverter(controller),
 			err:                           "the resource type 'invalid' is not valid for api version '2022-09-04'",
 		},
 		{
-			name: "syncidentityprovider - converter is nil",
+			name:            "syncidentityprovider - converter is nil",
+			ocmResourceType: "syncidentityprovider",
+			apiVersion:      "2022-09-04",
 			vars: map[string]string{
-				"ocmResourceType":           "syncidentityprovider",
-				"api-version":               "2022-09-04",
 				"resourceProviderNamespace": "microsoft.redhatopenshift",
 			},
 			syncIdentityProviderConverter: nil,
 			err:                           "the resource type 'syncidentityprovider' is not valid for api version '2022-09-04'",
 		},
 		{
-			name: "secret - resource type is valid",
+			name:            "secret - resource type is valid",
+			ocmResourceType: "secret",
+			apiVersion:      "2022-09-04",
 			vars: map[string]string{
-				"ocmResourceType":           "secret",
-				"api-version":               "2022-09-04",
 				"resourceProviderNamespace": "microsoft.redhatopenshift",
 			},
 			secretConverter: mock_api.NewMockSecretConverter(controller),
 			err:             "",
 		},
 		{
-			name: "secret - resource type is invalid",
+			name:            "secret - resource type is invalid",
+			ocmResourceType: "invalid",
+			apiVersion:      "2022-09-04",
 			vars: map[string]string{
-				"ocmResourceType":           "invalid",
-				"api-version":               "2022-09-04",
 				"resourceProviderNamespace": "microsoft.redhatopenshift",
 			},
 			secretConverter: mock_api.NewMockSecretConverter(controller),
 			err:             "the resource type 'invalid' is not valid for api version '2022-09-04'",
 		},
 		{
-			name: "secret - converter is nil",
+			name:            "secret - converter is nil",
+			ocmResourceType: "secret",
+			apiVersion:      "2022-09-04",
```

Comment:
nit: the apiVersion in these tests doesn't change, do we really need it as a test param?

Reply:
to be fair  I didn't even look at those, just applied a macro to get them out of the map :D

Reply:
refactoring at its best: macro+passing tests = win